cloudigrade/api/tests/views/test_dailyconcurrentusageviewset.py

```diff
@@ -1,229 +1,25 @@
 """Collection of tests for AccountViewSet."""
 import datetime
-from unittest.mock import patch
 
-import faker
 from django.test import TransactionTestCase
 from django.utils.translation import gettext as _
-from rest_framework.test import APIClient, APIRequestFactory
+from rest_framework.test import APIClient
 
-from api.models import ConcurrentUsageCalculationTask
 from api.tests import helper as api_helper
+from api.tests.views.shared.test_dailyconcurrentusageviewset import (
+    SharedDailyConcurrentUsageViewSetTest,
+)
 from util.misc import get_today
-from util.tests import helper as util_helper
 
 
-class DailyConcurrentUsageViewSetTest(TransactionTestCase):
+class DailyConcurrentUsageViewSetTest(
+    SharedDailyConcurrentUsageViewSetTest, TransactionTestCase
+):
```

Comment:
Since `SharedDailyConcurrentUsageViewSetTest` already extends `TransactionTestCase`, this shouldn't be necessary.

```suggestion
class DailyConcurrentUsageViewSetTest(SharedDailyConcurrentUsageViewSetTest):
```